src/ultimateterm.c

```diff
@@ -187,7 +186,9 @@ void term_displayheader(void) {
 
 void putstring_ascii(char *str) {
 	for (; *str; str++)
-		if (*str != LF) *str==BELL ? term_bell() : putchar(ascToPet[*str]);
+		if (*str != LF) {
```

Comment:
It didn't notice that before. Removing this code, bell will stop ringing in ASCII mode!

Reply:
yeah, youre right.  Ive reverted the term_bell function for that purpose for now.  As bad as duplicated code is, its very small.  And, we can use the bell function for other things as well, such as connection notification or termination notification.

Reply:
We can avoid code duplication calling bell from within asm:
https://github.com/xlar54/ultimateii-dos-lib/pull/35#discussion_r268690176

Reply:
asm("cmp #$07");
        asm("bne %g", skipbell);
        asm("jsr %v", term_bell)
    skipbell:
        asm("jsr $ffd2");